pyproject.toml

```diff
@@ -36,12 +36,12 @@ requests = "~2.28.1"
 scout-apm = "~2.26.1"
 sentry-sdk = "~1.5.12"
 wagtail = "~4.0.2"
-wagtail-airtable = "~0.2.1"
-wagtail-content-import = "~0.6.0"
+wagtail-airtable = "~0.2.1"  # This version isn't actually compatible with Wagtail 4 - https://torchbox.monday.com/boards/1123315381/pulses/1130533301
```

Comment:
Best to drop itdue to the notes you added.

suggestion: let's avoid Torchbox-specific links that others cannot access. Those could go in a wiki page on the repo

Reply:
doing some 🕵️‍♂️ work and will get back to this

Reply:
discussed and these all should be removed: `wagtail-airtable`, `wagtail-content-import`, `wagtail-image-import`, `wagtail-transfer`.

`wagtailaltgenerator` seems useful so we could leave it